StripeApplePay/StripeApplePay/Source/PaymentsCore/API/PaymentIntent+API.swift

```diff
@@ -78,7 +78,13 @@ extension StripeAPI.PaymentIntent {
                 .paramsByAddingTelemetryFields(toParams: pmAdditionalParams)
         }
 
-        apiClient.post(resource: endpoint, object: paramsWithTelemetry, completion: completion)
+         let parameters: [String: String] = [
+            "use_stripe_sdk": "true",
+            "payment_method": params.paymentMethod ?? "",
```

Comment:
Should we consider adding validation or handling for the `paymentMethod` and `returnURL` parameter before assigning a default empty string? Is it acceptable for this field to be empty, or should we enforce a valid payment method at this stage to prevent potential issues in the API call?